tests/test_workspaces/test_apis/test_export_settings/test_triggers.py

```
@@ -91,3 +91,23 @@ def test_post_save_workspace_general_settings_export_trigger_3(mocker, db):
+    # payload for the export trigger
+    workspace_general_settings_payload = {'reimbursable_expenses_object': 'JOURNAL ENTRY', 'corporate_credit_card_expenses_object': 'JOURNAL ENTRY'}
+    workspace_id = 5
+
+    Workspace.objects.filter(id=workspace_id).update(fyle_currency='YOLO')
+    QBOCredential.objects.filter(workspace_id=workspace_id).update(currency='USD')
+
+    export_trigger = ExportSettingsTrigger(workspace_general_settings=workspace_general_settings_payload, workspace_id=workspace_id)
+    export_trigger.post_save_workspace_general_settings()
+
+    workspace_general_setting = WorkspaceGeneralSettings.objects.filter(workspace_id=5).first()
+    assert workspace_general_setting.is_multi_currency_allowed == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if ...:` for truth checks

Replace comparison

(E712)